app/utils/computed-helpers.js

```diff
@@ -41,8 +41,8 @@ export const computedSegmentedLink = function(property) {
     },
     set(key, value) {
       const finalLink = values(value).join('');
-
-      if (finalLink && isValidUrl(finalLink.trim())) {
+      const isUserNamePresent = finalLink.split('.com/')[1];
```

Comment:
I don't understand. Why need to add more parsing? Don't we already have a separation between prefix and user-entered value? This is more error-prone. What if the prefix is `google.in`? Or `instagram.com/@`?

Reply:
Else it will save `https://facebook.com` in DB.

Reply:
right now all platforms we have defined have `.com`, We do need a better social link validator here though. I will search if I can find something useful.

or an alternative is set the prefix to 'https://' if `this.adress` is `null`

Reply:
I mean don't we have a separate variable for only the text which user has input? I believe that is how we join the prefix and user input

Reply:
updated 👍🏻